RxSwift/ObservableType+Extensions.swift

```diff
@@ -137,5 +116,17 @@ extension Hooks {
             _defaultErrorHandler = newValue
         }
     }
+    
+    /// Subscription callstack handler to fetch custom callstack information.
+    public static var subscriptionCallstackGenerationHandler: SubscriptionCallstackGenerationHandler {
```

Comment:
This isn't the callstack though, but the handler to generate the callstack at subscribe time. I was following the naming convention used for `defaultErrorHandler` since this provides a similar hook mechanism.

Reply:
Hi @benhjerrild ,

I'm not sure how to express myself in the spirit of english language, but I can provide the reasons why certain things were named the way they have.

Hooks - customization points
defaultErrorHandler - If no custom error handling is defined, handle the error in this way.

To me, it makes most sense to call something a handler if it has type `T` -> (). subscriptionCallstackGenerationHandler is of type `() -> [String]`. Maybe `recordCallstack` or something similar would be more appropriate.

Reply:
@kzaher OK, thanks for clarifying. In that case I think the first name you proposed makes more sense. I pushed a commit to update this.